pandas/tests/generic/test_frame.py

```diff
@@ -287,3 +289,36 @@ def test_deepcopy_empty(self):
         empty_frame_copy = deepcopy(empty_frame)
 
         self._compare(empty_frame_copy, empty_frame)
+
+    def test_datetimeField_after_setitem(self):
```

Comment:
Mention the use of `at` in the title: `test_datetime_setitem_with_at`

Reply:
Sure